.teamcity/settings.kts

```diff
@@ -45,6 +45,7 @@ project {
 
 	params {
 		param("env.NODE_OPTIONS", "--max-old-space-size=32000")
+		param("use_cached_node_modules", "true")
```

Comment:
Setting it as `text` will allow us to add a meaningful name and description. 

There is also [`checkbox` ](https://teamcity.jetbrains.com/app/dsl-documentation/jetbrains.build-server.configs.kotlin.v2019_2/-parametrized-with-type/checkbox.html) that may be more appropriate for a boolean.

We can as well leave this change for a future PR and clean up all the params, many of them could benefit from changing to `checkbox`, up to you.

Reply:
yeah, that sounds good to me!